### PR TITLE
TL Localization - Chapter 6, section 2 - part 1 (1,146 words)

### PR DESCRIPTION
**Pull request by heywens**
Word count for this contribution does not include tags, codes, link, etc.
This is a partial translation of Chapter 6, section 2 which contains 420 lines with an estimate of  3.5K+ translatable words (file has 4,154 words, including tags, codes, link, etc).

### CODE REVIEW

File: book/06-github/sections/2-contributing.asc [hunk 1]
```diff
@@ -1,78 +1,78 @@
-=== Contributing to a Project
+=== Pag-aambag sa isang Proyekto
 
-Now that our account is set up, let's walk through some details that could be useful in helping you contribute to an existing project.
```
Comment (jjrang11) [inline book/06-github/sections/2-contributing.asc:3]: kindly change na-setup to na-set up ? 

As setup is a noun and set up is a verb :)

File: book/06-github/sections/2-contributing.asc [hunk 2]
```diff
@@ -1,78 +1,78 @@
...
+Narito kung paano ito gumagana:
 
-1. Fork the project
-2. Create a topic branch from `master`.
-3. Make some commits to improve the project.
-4. Push this branch to your GitHub project.
-5. Open a Pull Request on GitHub.
-6. Discuss, and optionally continue committing.
-7. The project owner merges or closes the Pull Request.
+1. I-fork ang proyekto
```
Comment (jjrang11) [inline book/06-github/sections/2-contributing.asc:39]: `Magbukas ng isang Kahilingan ng Pull sa GitHub.` to `Magbukas ng isang Pull na Kahilingan sa GitHub.`

File: book/06-github/sections/2-contributing.asc [hunk 3]
```diff
@@ -1,78 +1,78 @@
...
+7. Pagsasamahin o isasara ng may-ari ng proyekto ang Kahilingan ng Pull.
 
-This is basically the Integration Manager workflow covered in <<_distributed_git#_integration_manager>>, but instead of using email to communicate and review changes, teams use GitHub's web based tools.
+Ito ang karaniwan na workflow ng Tagapamahala ng Paglagom na tinalakay sa <<_distributed_git#_integration_manager>>, ngunit sa halip na gagamit ng email sa pakikipag-ugnayan at pagsusuri ng mga pagbabago, ang pangkat ay gumagamit ng mga kagamitan ng GitHub na nakabatay sa web.
 
-Let's walk through an example of proposing a change to an open source project hosted on GitHub using this flow.
+Talakayin natin ang isang halimbawa ng pagpapanukala ng pagbabago sa isang open-source na proyekto na naka-host sa GitHub gamit ang daloy na ito.
 
-===== Creating a Pull Request
+===== Paglilikha ng isang Kahilingan ng Pull
```
Comment (jjrang11) [inline book/06-github/sections/2-contributing.asc]: `Paglilikha ng isang Kahilingan ng Pull ` to `Paglikha ng isang Pull na Kahilingan`

**Review by jjrang11 — COMMENTED** [review]
Hello @heywens I've suggested a few changes for your translation :)

**Comment by heywens** [comment]
hey @jjrang11, thanks for the review. i've applied necessary changes based on your feedback.

**Pull request merged**